### PR TITLE
WAZO-2505: `calld` events

### PR DESCRIPTION
**Pull request by ja0x**
Depends-On: https://github.com/wazo-platform/xivo-bus/pull/59

### CODE REVIEW

**Comment by ja0x** [comment]
recheck

File: integration_tests/suite/test_meetings.py [hunk 1]
```diff
@@ -288,7 +288,7 @@ def test_participant_joins_sends_event(self):
                 name='meeting',
             ),
         )
-        bus_events = self.bus.accumulator(f'meetings.{meeting_uuid}.participants.joined')
+        bus_events = self.bus.accumulator(headers={'meeting_uuid': meeting_uuid})
```
Comment (pc-m) [inline integration_tests/suite/test_meetings.py:291]: I don't remember why we chose to use `user_uuid:<UUID>: true`. Should we use the same pattern for meetings?

File: integration_tests/suite/test_switchboards.py [hunk 2]
```diff
@@ -463,8 +458,12 @@ def test_given_one_queued_call_and_one_operator_when_answer_then_operator_is_bri
         self.confd.set_switchboards(MockSwitchboard(uuid=switchboard_uuid))
         self.confd.set_users(MockUser(uuid=user_uuid, line_ids=[line_id]))
         self.confd.set_lines(MockLine(id=line_id, name='switchboard-operator/autoanswer', protocol='test'))
-        routing_key = 'switchboards.{uuid}.calls.queued.updated'.format(uuid=switchboard_uuid)
-        bus_events = self.bus.accumulator(routing_key)
+        bus_events = self.bus.accumulator(
+            headers={
+                'switchboard_uuid': switchboard_uuid,
```
Comment (pc-m) [inline integration_tests/suite/test_switchboards.py:463]: Same thing as the `meeting_uuid` above

**Review by pc-m — APPROVED** [review]

**Comment by ja0x** [comment]
recheck

**Comment by ja0x** [comment]
recheck

**Pull request merged**